packages/yoga/src/Banner/native/Banner.jsx

```diff
@@ -78,7 +78,7 @@ const Banner = forwardRef((props, ref) => {
             marginRight="xxsmall"
           />
         )}
-        <Text.Small flex={1} marginVertical="xxsmall">
```

Comment:
@naabraz it seems we have an issue using it at native components. 

![Screenshot 2023-08-01 at 12 36 10](https://github.com/gympass/yoga/assets/13206745/0c59dceb-39ec-48e6-8c85-a446026b4bca)

Actually, I also could experience the warnings pointed out above as runtime errors (with slightly different messages) on the simulator, which strangely didn't happen testing again here (just warnings on the console 😐). But to prevent any possibility I thought it best to avoid using it. I also noticed that no other native component is using this prop.

Interestingly, using `mv=` doesn't offer any problems, as do the `marginTop` and `marginBottom` props.

Reply:
Ok, thanks for the explanation! 

I see that on `end-user-native` only a few components are using as `marginVertical`, most already use it as `mv`.